### DISCUSSION TITLE
I can't start a flight with the A32NX

### DISCUSSION THREAD

**Discussion by DedaL54** [flybywiresim/aircraft#7308] [Help]
I have no choice, I have to ask for help. With the new installer I installed the A32NX stable version.  When I start a flight at a given airport there is no way for that to happen. Either I wait a very long time or I turn off the desktop. I tried everything I read as advice. I also tried with a completely empty Community folder, deleted and installed ... it doesn't work!
Everything else, all Asobo models and all addons work great . The only thing I can't do is run the A32NX in either the stable or the experimental version. I have no idea what to do anymore so here I am asking for advice if anyone might have.
Of course, I have a problem after the last update! I don't know why it doesn't accept flybywire A32NX ?!
Thanks in advance if anyone can help!

**Comment by Saschl** [comment]
Hi, can you please open the content manager in MSFS and search for 'flybywire'. Then please take a screenshot and put it here.

**Reply by Benjozork** [reply]
Just so you know, you posted the screenshot as a separate answer so this is why @Saschl did not see it. You should post it as a response like you did here.

**Reply by DedaL54** [reply]
Yes, I’m aware of the mistake but I didn’t think it was that important!

**Reply by DedaL54** [reply]
<a target="_blank" href="https://fotkica.com/slike.php?slika=1_103522525_Untitled.png"><img src="https://fotkica.com/thumbs3/1_tmb_103522525_Untitled.png" border="0" /></a>

Thank you for attention ! :)

**Reply by Benjozork** [reply]
Are you able to show me your community folder ?

**Comment by DedaL54** [comment]


**Reply by Benjozork** [reply]
Just FYI, our main platform for helping users out is Discord. I understand this is not stated explicitly but GitHub Discussions is not used actively at the moment.

**Reply by DedaL54** [reply]
Okay, thanks for the info. I'll have to open a Discord account! Thank you !
